sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cell::RefCell;
+use std::collections::HashMap;
+use std::process::Child;
+use std::rc::Rc;
+
+use crate::io::emu::{EmuState, EmuValue, Emulator};
+use crate::transport::ti50emulator::{Inner, Ti50Emulator};
+use crate::transport::{Result, TransportError};
+
+/// Structure representing `Emulator` sub-process based on TockOS host-emulation architecture.
+pub struct Ti50SubProcess {
+    /// reference to internal states of Ti50Emulator
+    _inner: Rc<RefCell<Inner>>,
```

Comment:
Why do these have leading underscores?

Reply:
To avoid warning - unused variable.

Reply:
Right, but if it's never read, why is it here? Those warnings are emitted for a reason.